frontend/src/metabase/services.js

```diff
@@ -536,4 +536,7 @@ export const ActionsApi = {
   delete: POST("/api/action/row/delete"),
   bulkUpdate: POST("/api/action/bulk/update/:tableId"),
   bulkDelete: POST("/api/action/bulk/delete/:tableId"),
+  execute: POST(
+    "/api/dashboard/:dashboardId/dashcard/:dashcardId/action/:actionId/execute",
+  ),
```

Comment:
@snoe or @metamben what happens if we pass the wrong action id here? i.e. an action id that doesn't match the action id that is actually linked to the dashcard?

Reply:
@iethree a 404 happens because no entry in `report_dashboardcard` will be found in the DB with those IDs.